File: src/cs/ifelse.cs
class Program
{
	public static void Main()
	{
		var value = true;
        if (value)
        {
		    System.Console.WriteLine("Got true");
        }
        else
        {
            System.Console.WriteLine("Got else");
        }
		
        value = false;
        if (value)
		    System.Console.WriteLine("Got true");
        else
            System.Console.WriteLine("Got else");

        var result = 3;
        if (result == 1)
            System.Console.WriteLine("Got true");
        else if (result == 2)
            System.Console.WriteLine("Got else");
        else
            System.Console.WriteLine("Got if else");
	}
}

File: src/vb/ifelse.vb
Public Module Program
    Sub Main()
        Dim value as Boolean = True
        If value Then
		    System.Console.WriteLine("Got true")
        Else
            System.Console.WriteLine("Got else")
        End If
		
        value = False
        If value
		    System.Console.WriteLine("Got true")
        Else
            System.Console.WriteLine("Got else")
        End If

        Dim result As Integer = 3
        if result = 1
            System.Console.WriteLine("Got true")
        ElseIf result = 2
            System.Console.WriteLine("Got else")
        Else
            System.Console.WriteLine("Got if else")
        End If
    End Sub
End Module